Source/KSCrash/Reporting/Filters/KSCrashReportFilterAppleFmt.m

```diff
@@ -496,19 +530,22 @@ - (NSString*) binaryImagesStringForReport:(NSDictionary*) report
             NSString* path = [image objectForKey:@KSCrashField_Name];
             NSString* name = [path lastPathComponent];
             NSString* uuid = [self toCompactUUID:[image objectForKey:@KSCrashField_UUID]];
-            NSString* isBaseImage = (path && [executablePath isEqualToString:path]) ? @"+" : @" ";
-
+            //NSString* isBaseImage = (path && [executablePath isEqualToString:path]) ? @"+" : @" ";
+            NSString* isBaseImage = @"";
```

Comment:
Please remove this argument from format instead of using ""

Reply:
Thanks. I will fix this!